torch/testing/_internal/common_device_type.py

```diff
@@ -877,6 +877,20 @@ def skipCUDAIfRocm(fn):
 def skipCUDAIfNotRocm(fn):
     return skipCUDAIf(not TEST_WITH_ROCM, "test doesn't currently work on the CUDA stack")(fn)
 
+# Skips a test for specified CUDA version, given in [major, minor] form.
+def skipCUDAVersion(version : [int] = None):
```

Comment:
```suggestion
def skipCUDAIfVersionIn(version : [Tuple[int, int]] = None):
```
this sounds better --> e.g. you can do 
```
@skipCUDAIfVersionIn([(11,2), (11,1)])
```

Reply:
and obviously version shouldn't be a list but a tuple2